test_regress/t/t_dpi_arg_inout_type__Dpi.out

```diff
@@ -10,8 +10,8 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
- 
- 
+
+
```

Comment:
Thanks for fixing the extra spaces - can you please throw that into a separate pull request and merge it into master - that'll cut down the number of files changing. Thanks.

Reply:
Done